From 03dd8356ae60805407582136e67d19876ccc47c1 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 2 Jul 2012 16:58:57 +0800
Subject: [PATCH 1703/2508] ENGR00215531-4 V4L2:OV5640_mipi:Correct sensor
 data attribute

This patch corrects sensor data struct by adding static
attribute.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/media/video/mxc/capture/ov5640_mipi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ov5640_mipi.c b/drivers/media/video/mxc/capture/ov5640_mipi.c
index 07c3e81..bc2e480 100644
--- a/drivers/media/video/mxc/capture/ov5640_mipi.c
+++ b/drivers/media/video/mxc/capture/ov5640_mipi.c
@@ -82,7 +82,7 @@ struct ov5640_mode_info {
 /*!
  * Maintains the information on the current state of the sesor.
  */
-struct sensor_data ov5640_data;
+static struct sensor_data ov5640_data;
 
 static struct reg_value ov5640_init_setting_30fps_VGA[] = {
 
-- 
1.7.7.4

